### ISSUE TITLE
💡 [REQUEST] - Can choose a language at the launch layout

### ISSUE THREAD

**Issue by SharkyFlou** [alvarohghg/GymQuest#15]
### Summary

The user can choose the language in which the application is displayed.
There could be English and French at the beginning of the implementation.

I propose this new feature because I think I can implement it.

### Basic Example

//

### Drawbacks

This would allow the application to be used and understood by people who do not speak English.

**Comment by AlexLunaP** [comment]
Nice idea!
You can start working on this. Let us know your updates.

Maybe you can add something like a settings icon in the login screen, to switch between languages?

**Comment by SharkyFlou** [comment]
Yes this was the idea I had in mind, I start working on it now !
